test/environment.js

```diff
@@ -22,7 +22,7 @@ const auth = {
 };
 
 // Default test interval is 5 minutes
```

Comment:
In this comment series https://github.com/openshift/openshift-ansible/pull/7880/files#r180740741 it was suggested we accept a `time.Duration` string.  That probably makes sense, with a check for a min value.

Reply:
I think that means something like this:https://golang.org/pkg/time/#example_Duration

    func ParseDuration

    func ParseDuration(s string) (Duration, error)
    ParseDuration parses a duration string. A duration string is a possibly signed sequence of decimal numbers, 
    each with optional fraction and a unit suffix, such as "300ms", "-1.5h" or "2h45m". Valid time units are "ns", "us" 
    (or "µs"), "ms", "s", "m", "h".




I bet that this npm package might cover it.  https://www.npmjs.com/package/parse-duration

We could spin off as a separate issue if you want to merge this PR.

Reply:
Lets do that as followUp :)

Reply:
created #27 for this